mkdocsdrawioexporter/tests/exporter.py

```diff
@@ -233,12 +233,36 @@ def test_export_file_logs_exc_on_raise(self, call_mock):
         self.log.exception = MagicMock()
 
         result = self.exporter.export_file(
-                source, 0, dest, drawio_executable, 'svg')
+                source, 0, dest, drawio_executable, [], 'svg')
 
         assert result == None
         self.log.exception.assert_called_once()
         call_mock.assert_called_once()
 
+    @patch('subprocess.call')
+    def test_export_file_honours_drawio_args(self, call_mock):
+        source = sep + join('docs', 'diagram.drawio')
+        dest = sep + join('docs', 'diagram.drawio-0.svg')
+        drawio_executable = sep + join('bin', 'drawio')
+
+        def call_with_drawio_args(drawio_args):
+            self.exporter.export_file(
+                    source, 0, dest, drawio_executable, drawio_args, 'svg')
+
+        cmd = [
+            drawio_executable,
+            '--export', source,
+            '--page-index', '0',
+            '--output', dest,
+            '--format', 'svg',
+        ]
```

Comment:
The duplication here is a little ugly, I'll try to extract these values or just move the assertions and `cmd` value into `call_with_drawio_args()`.